.github/workflows/linux.yml

```diff
@@ -22,3 +22,72 @@ jobs:
         with:
           name: ooniprobe-linux-amd64
           path: ./CLI/linux/amd64/ooniprobe
+
+  armhf_build:
+    runs-on: "ubuntu-latest"
+    name: Build on armhf Buster
+    steps:
+      - uses: actions/checkout@v2.1.0
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Build ooniprobe
+        id: runcmd
+        with:
+          arch: armv7
+          distro: buster
+          # distro: ubuntu20.04 # no linux-headers-arm64
+
+          # Not required, but speeds up builds by storing container images in
+          # a GitHub package registry.
+          githubToken: ${{ github.token }}
+
+          run: |
+            uname -a
+            apt-get -qq update
+            apt-get install -qq -y --no-install-recommends gcc git linux-headers-armmp wget musl-dev ca-certificates libc6-dev
```

Comment:
Why installing `musl-dev` here? I admit not knowing much about what `musl-dev` could do on Debian, but I suspect that, if we want the compiler to link against it, then we need other compiler flags. (Am I missing something?)